clients/banking/src/components/SandboxUserPicker.tsx

```diff
@@ -0,0 +1,331 @@
+    transitionProperty: "background-color",
+  },
+  activeItem: {
+    backgroundColor: colors.gray[50],
+  },
+  hoveredItem: {
+    backgroundColor: colors.gray[50],
+  },
+  pressedItem: {
+    backgroundColor: colors.gray[100],
+  },
+  selected: {
+    color: colors.gray[800],
+  },
+  itemText: {
+    overflow: "hidden",
+    whiteSpace: "nowrap",
+    textOverflow: "ellipsis",
+  },
+  value: {
+    justifyContent: "center",
+    height: 40,
+    paddingHorizontal: spacings[24],
+    transitionProperty: "background-color",
+    transitionDuration: "150ms",
+  },
+  selectListTitle: {
+    paddingHorizontal: spacings[24],
+    paddingVertical: spacings[8],
+  },
+});
+
+const PER_PAGE = 20;
+
+type ItemProps = {
+  firstName?: string | null;
+  lastName?: string | null;
+  onPress: () => Future<unknown>;
+  isActive: boolean;
+};
+
+const Item = ({ onPress, isActive, firstName, lastName }: ItemProps) => {
+  const [isLoading, setIsLoading] = useState(false);
+  return (
+    <Pressable
+      role="listitem"
+      onPress={() => {
+        setIsLoading(true);
+        onPress().tap(() => setIsLoading(false));
+      }}
+      style={({ hovered, pressed }) => [
+        styles.value,
+        styles.item,
+        hovered && styles.hoveredItem,
+        pressed && styles.pressedItem,
+        isActive && styles.activeItem,
+      ]}
+    >
+      <Box direction="row" alignItems="center">
+        <LakeText
+          variant="smallRegular"
+          color={colors.gray[900]}
+          numberOfLines={1}
+          style={[styles.itemText, isActive && styles.selected]}
+        >
+          {firstName} {lastName}
+        </LakeText>
+
+        <Fill minWidth={12} />
+
+        {isActive ? (
+          <Icon color={colors.positive[500]} name="checkmark-filled" size={16} />
+        ) : isLoading ? (
+          <ActivityIndicator color={colors.gray[500]} size={16} />
+        ) : (
+          <Space width={16} />
+        )}
+      </Box>
+    </Pressable>
+  );
+};
+
+type PickerContentsProps = {
+  onEndorse: () => void;
+};
+
+export const SandboxUserPickerContents = ({ onEndorse }: PickerContentsProps) => {
+  const [sandboxUsers, { setVariables }] = useQuery(SandboxUsersDocument, {
+    first: PER_PAGE,
+    orderBy: {
+      field: "createdAt",
+      direction: "Asc",
+    },
+  });
+  const [endorseSandboxUser] = useMutation(EndorseSandboxUserDocument);
+
+  return match(sandboxUsers)
+    .with(AsyncData.P.NotAsked, () => null)
+    .with(AsyncData.P.Loading, () => (
+      <LoadingView color={colors.current[500]} style={styles.loader} />
+    ))
+    .with(AsyncData.P.Done(Result.P.Error(P._)), () => null)
+    .with(AsyncData.P.Done(Result.P.Ok(P.select())), ({ sandboxUser, sandboxUsers }) => {
+      return (
+        <Box grow={1}>
+          <Connection connection={sandboxUsers}>
+            {sandboxUsers => (
+              <FlatList
+                role="list"
+                style={styles.list}
```

Comment:
Instead of wrapping everything in a `Box` with `grow={1}`, add `flexGrow: 1` to `styles.list`